sdcm/nemesis.py

```diff
@@ -4585,6 +4606,7 @@ def disrupt(self):
 class ValidateHintedHandoffShortDowntime(Nemesis):
     disruptive = True
     kubernetes = True
+    free_tier_set = True
```

Comment:
This nemesis is absent in the list you posted here: https://github.com/scylladb/scylla-cluster-tests/pull/5504#issuecomment-1337171769
Which one is correct?

Reply:
I created list automatically as it is seen by SCT - these nemesis are absent because they are disabled due issues. 
Disruption method names are not starting with `self.disrupt_` so SCT does not run them in random monkey tests.